packages/datatrak-web/src/layout/UserMenu/UserInfo.tsx

```diff
@@ -3,19 +3,47 @@
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import { Typography } from '@material-ui/core';
-import { Button, RouterLink } from '@tupaia/ui-components';
-import { useUser } from '../../api/queries';
+import { RouterLink } from '@tupaia/ui-components';
+import { Button } from '../../components';
+import { useProjects, useUser } from '../../api/queries';
 import { MOBILE_BREAKPOINT, ROUTES } from '../../constants';
+import { ProjectSelectModal } from './ProjectSelectModal';
 
 const Wrapper = styled.div`
   @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
     display: none;
   }
 `;
 
+const Details = styled.div`
+  display: flex;
+  align-items: center;
+
+  > span {
+    color: ${props => props.theme.palette.text.primary};
+    position: relative;
+    top: -1px;
+    font-size: 1.2em;
+    margin-left: 0.5rem;
+  }
+`;
+
+const ProjectButton = styled(Button).attrs({
+  variant: 'text',
+})`
+  color: ${props => props.theme.palette.text.secondary};
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+
+  &:hover {
+    color: #2a78c3;
```

Comment:
I wonder if we need to check with Flic if this is meant to be the primary blue?

Reply:
In the design it is labelled as hover blue so I've added it to the theme as a hover color